packages/salesforce-adapter/test/adapter.crud.test.ts

```diff
@@ -1039,6 +1039,29 @@ describe('SalesforceAdapter CRUD', () => {
         })
       })
 
+      describe('when the request has errorMessage', () => {
+        beforeEach(async () => {
+          connection.metadata.deploy.mockReturnValueOnce(mockDeployResult({
```

Comment:
```suggestion
          connection.metadata.deploy.mockReturnValue(mockDeployResult({
```

`mockReturnValue` is prefered to avoid issues between parallel test runs.